widgets/google-map/google-map.php

```diff
@@ -441,10 +450,34 @@ function get_settings_form() {
 					'<a href="https://developers.google.com/maps/documentation/javascript/get-api-key" target="_blank" rel="noopener noreferrer">',
 					'</a>'
 				)
+			),
+
+			'responsive_breakpoint' => array(
+				'type'        => 'number',
+				'label'       => __( 'Responsive breakpoint', 'so-widgets-bundle' ),
+				'default'     => '780',
+				'description' => __( 'This setting controls when the map will use the mobile zoom. This breakpoint will only be used if a mobile zoom is set in the SiteOrigin Google Maps settings. The default value is 780px', 'so-widgets-bundle' )
 			)
 		);
 	}
 
+	function modify_instance( $instance ) {
+		if ( empty( $instance['settings']['mobile_zoom'] ) ) {
```

Comment:
@braamgenis I'm unsure why this check would trigger the error you're getting. 

This same check is done with the contact form and that doesn't have an error... wait, do you get an error message with that widget?

https://github.com/siteorigin/so-widgets-bundle/blob/1.13.3/widgets/contact/contact.php#L686

Reply:
`$instance` can be an empty `array`. E.g. there are times when we run widgets' `form` functions with an empty `array` simply to enqueue their scripts.

Reply:
For future reference, should this be done with `modify_instance` in general?